src/commands/language/add.ts

```diff
@@ -27,6 +27,6 @@ export default class LanguageAdd extends Command {
 
         const response = await this.languageService.add(code);
 
-        this.log(response);
+        this.log(response.id);
```

Comment:
I think it would make sense if we added some text here. Would something like `${code} was added as a language` be better?